src/EnergyPlus/SimulationManager.cc

```diff
@@ -1123,6 +1131,11 @@ namespace SimulationManager {
             state.dataGlobal->DoWeathSim = true;
         }
 
+        if ((!state.dataGlobal->DoDesDaySim && !state.dataGlobal->DoWeathSim && !state.dataGlobal->DoHVACSizingSimulation) &&
+            (state.dataGlobal->DoZoneSizing || state.dataGlobal->DoSystemSizing)) {
+            state.dataGlobal->DoPureLoadCalc = true;
+        }
```

Comment:
Do we need to add a condition for DoPlantSizing here?
```suggestion
        if ((!state.dataGlobal->DoDesDaySim && !state.dataGlobal->DoWeathSim && !state.dataGlobal->DoHVACSizingSimulation) &&
            (state.dataGlobal->DoZoneSizing || state.dataGlobal->DoSystemSizing || state.dataGlobal->DoPlantSizing)) {
            state.dataGlobal->DoPureLoadCalc = true;
        }
```

Reply:
I don't believe it is needed.   The use case here doesn't really include doing only plant sizing.